app/templates/auth/login.html

```diff
@@ -1,5 +1,5 @@
 {% extends "base.html" %}
```

Comment:
I can't be sure, but when viewing this file in full it seems to have a massive amount of indentation. Much greater than the usual 4-space or 2-space. Can this be checked in your IDE?

Reply:
I think this might be related to the template using tabs instead of spaces? I have replaced them with the 4-spaces used everywhere. Should be fine now.